### ISSUE TITLE
Inversion Results

### ISSUE THREAD

**Issue by abcdvzz** [Vicky0522/I2VEdit#9]
Hi I only tried the edm inversion_video code, the reconstruction results are bad.

FYI, I used the code here: https://huggingface.co/docs/diffusers/en/using-diffusers/svd, and set param latents = inv_latents, I also changed the following code to just use image_latents and image_embeddings in the unet:

    # image_latents = torch.cat(
    #     [
    #      image_latents.unsqueeze(1).repeat(1, skip, 1, 1, 1),
    #      firstframe_latents.unsqueeze(1).repeat(1, latents.shape[1]-skip, 1, 1, 1)
    #     ],
    #     dim=1
    # )
    image_latents = image_latents.unsqueeze(1).repeat(1, latents.shape[1], 1, 1, 1)

Is there anything else I should notice? What about your results with only edm inversion?

**Comment by Vicky0522** [comment]
You can directly try EDM inversion using this repository by disabling other parameters and enabling inversion (`use_inversed_latents=true`) in the configuration files. Hope this will be helpful for you.
